sysutils/port-whatsnew/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                port-whatsnew
+version             0.1
+revision            0
+categories          sysutils macports
+maintainers         gwmail.gwu.edu:egall openmaintainer
```

Comment:
You'll want to fix this to conform to current standards (including adding the github ID.)

Reply:
ok, fixed in fe01049.